Makefile

```diff
@@ -152,10 +152,12 @@ LINUX_CMAKE_PARAMS = \
 clean: clean-android clean-osx clean-ios clean-rpi clean-tests clean-xcode clean-linux
 
 clean-android:
-	@cd android/ && \
-	./gradlew clean
 	rm -rf ${ANDROID_BUILD_DIR}
+	rm -rf android/build
+	rm -rf android/tangram/build
 	rm -rf android/tangram/libs
+	rm -rf android/demo/build
+	rm -rf android/demo/libs
```

Comment:
We should also remove files created for `android-debug` target.

I think the following:
```
android/demo/jni
android/*/libs
android/*/obj
*.mk
```

I do not have a device, which can run gdb (samsung galaxy s5 :(), so can not test this. @blair1618 your personal phone however can be used for debugging, can you check and update?

Thanks